src/app/core/guideline.service.ts

```diff
@@ -53,6 +53,18 @@ export class GuidelineService {
       });
   }
 
+  async getFrameworks(): Promise<FrameworkDocument[]> {
+    return this.http
+      .get<{total: number, results: FrameworkDocument[]}>(STANDARD_GUIDELINE_ROUTES.SEARCH_FRAMEWORKS({}))
+      .pipe(
+        retry(3),
+        catchError(this.handleError),
+        map((res) => {
+          return res.results.filter(framework => framework.name !== 'CAE CDE 2019');
```

Comment:
Since we have standard guidelines operational how would you feel about deleting CAE CDE 2019 so we don't need to filter it out? I'm 99.999% positive it was never mapped to anything but we can check.

Reply:
We can't delete these because there are currently 50 objects with outcomes mapped to the CAE CDE 2019 framework.  These range across the NCCP, Secure Coding, and NCyTE collections